CMakeLists.txt

```diff
@@ -21,7 +23,7 @@ option(BUILD_SHARED_LIBS "Build shared libraries" OFF)
 option(OATPP_DIR_SRC "Path to oatpp module directory (sources)")
 option(OATPP_DIR_LIB "Path to directory with liboatpp (directory containing ex: liboatpp.so or liboatpp.dynlib)")
 option(OATPP_BUILD_TESTS "Build tests for this module" ON)
-option(OATPP_INSTALL "Install module binaries" ON)
+option(OATPP_INSTALL "Install module binaries" OFF)
```

Comment:
Please remove changes to `CMakeLists.txt`

Reply:
Ohhh, sorry, my bad habits with `git commit -am` instead of `git commit -m`. Will fix it ASAP